REDME.md

```diff
@@ -0,0 +1,78 @@
+# My Final Project
+
```

Comment:
* please, name `REDME.md` correctly: `README.md` otherwise it doesn't show the information when I visit the repo